server/jobs/job_manager.go

```diff
@@ -23,12 +29,22 @@ type RegisteredJob struct {
 	isEnabledByConfig func(env mscalendar.Env) bool
 }
 
+var scheduleFunc = func(api cluster.JobPluginAPI, id string, wait cluster.NextWaitInterval, cb func()) (io.Closer, error) {
```

Comment:
I do not get why are you declaring this function here, instead of using directly `cluster.Schedule(api, id, wait, cb) whenever you need it.

Reply:
It's so the test can mock the function. I can instead make it into an interface, but since the `cluster` package exposes it as a flat function, I chose this method instead.